### DISCUSSION TITLE
Directus 9 with Nginx and Https

### DISCUSSION THREAD

**Discussion by ddodson1** [directus/directus#9699] [Archive]
Hello,

While running our Directus 9 instance with 'npx directus start' and accessing it at http://{{server_ip}}:8055 we are having no issues logging into the admin app instance and accessing all of our data. However,  proxying the traffic through Nginx using ssl is causing us to receive a 'No App Access' error in the admin app.

![Screen Shot 2021-11-10 at 9 38 52 AM](https://user-images.githubusercontent.com/94061070/141133206-674f9524-b064-4c68-b115-83e137c6e052.png)

There are no issues using nginx and https with the Directus API, the only issues are with accessing the admin app.

```
    listen [::]:443 ssl;
    listen 443 ssl;
    ...
       certificate info
    ...

    location / {
                proxy_redirect off;
                proxy_pass http://127.0.0.1:8055$uri;
                proxy_http_version 1.1;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection 'upgrade';
                proxy_set_header Host $host;
                proxy_cache_bypass $http_upgrade;
        }
```

This is the Nginx configuration we have set up at the moment which seems to load directus fine in the browser at our domain until we login and get the app access error.

**Comment by salohcin714** [comment]
@ddodson1 

Your `proxy_pass` value looks incorrect. Try `proxy_pass http://127.0.0.1:8055$request_uri;` and see if that fixes your problem.
